_aicspylibczi/Reader.cpp

```diff
@@ -118,7 +120,7 @@ namespace pylibczi {
       bool regularShape = true;
       for (int i = 1; regularShape && i<dShape_.size(); i++) {
           for (auto kVal: dShape_[i]) {
-              if( kVal.first == DimIndex::S ) continue;
+              if (kVal.first==DimIndex::S) continue;
```

Comment:
it looks like this file didn't get run through the formatter?

Reply:
Actually it did. I just ran it through the JetBrains default but it doesn't appear to be as specific as clang-format. I'll find a clang-format file that is decent and try applying that.

Reply:
are these consistent whitespace?  `if (!isMosaic() && (!sceneBool || sceneSize==1))`  vs `if (kVal.first==DimIndex::S)` ? If so, then I'm good.

Reply:
So I added a .clang-format config with Mozilla presets. Apparently this is Mozilla formatted. Logical operations like == != etc No spaces but && and || are always padded with spaces. So yes it's a consistent style.